src/Weft/Generics/PprQuery.hs

```diff
@@ -52,7 +48,9 @@ instance ( KnownSymbol name
   gPprQuery (M1 (K1 (Just (args, rec)))) =
     sep
       [ text (symbolVal $ Proxy @name) <> pprArgs args
-      , pprQuery rec
+      , char '{'
+      , nest 4 $ pprQuery rec
```

Comment:
what does `nest 4` do / why 4 ?

Reply:
Indents the subrecord with 4 spaces. Chosen arbitrarily, but since this is a pretty printer that seems reasonable to me.